amd_openvx_extensions/amd_media/decoder.cpp

```diff
@@ -163,6 +165,7 @@ static inline int num_hw_devices() {
 
 static int hw_decoder_init(AVCodecContext *ctx, const enum AVHWDeviceType type, AVBufferRef *hw_device_ctx, int hw_device_id)
 {
+    //printf("hw_decoder_init called for device_id:%d\n", hw_device_id);
```

Comment:
please remove this comment, if it is not needed.

Reply:
done